src/contexts/data/likesContext.js

```diff
@@ -0,0 +1,38 @@
+// import axios from "axios";
```

Comment:
if you have done this intentionally you can add a note explaining the reason,it'll help the reviewer.